api-examples/pytorch_cpp/classify-text.cpp

```diff
@@ -0,0 +1,133 @@
+#include <torch/script.h>
+#include "json.hpp"
+using json = nlohmann::json;
+
+#include <iostream>
+#include <fstream>
+#include <memory>
+#include <string>
+#include <algorithm>
+#include <glob.h>
+
+#define UNK 3
+#define PAD 0
+
+
+void lower_case(const std::string& word, std::string& dst) {
+    dst.resize(word.size());
+    std::transform(
+        word.begin(),
+        word.end(),
+        dst.begin(),
+        ::tolower
+    );
+}
+
+
+void find_vocab_file(const std::string& bundle, const std::string& vocab, std::string& dst) {
+    glob_t glob_result;
```

Comment:
not a huge deal, but glob is POSIX, not portable

Reply:
I added a mention of this in the README